### PR TITLE
Element-wise tests now use or seeded with hypothesis

### PR DESCRIPTION
**Pull request by jspisak**

### CODE REVIEW

**Comment by houseroad** [comment]
@caffe2bot test this please

File: caffe2/python/operator_test/elementwise_ops_test.py [hunk 1]
```diff
@@ -105,8 +107,8 @@ def powt_grad(g_out, outputs, fwd_inputs):
                                    grad_reference=powt_grad)
 
 
-    @given(n=st.integers(5, 6), m=st.integers(4, 6), **hu.gcs)
-    def test_sqr(self, n, m, gc, dc):
+    @given(n=st.integers(5, 6), m=st.integers(4, 6), seed=st.integers(0, 1000), **hu.gcs)
+    def test_sqr(self, n, m, gc, dc, seed):
```
Comment (houseroad) [inline caffe2/python/operator_test/elementwise_ops_test.py:111]: We also need to set the seed in this case. :-)

**Review by houseroad — COMMENTED** [review]
Good job, @jspisak!
Don't forget to set the seed in test_sqr case :-)

**Comment by jspisak** [comment]
@houseroad - if you will sir.. :)

**Comment by jspisak** [comment]
@caffe2bot test this please

**Comment by houseroad** [comment]
@caffe2bot retest this please

**Review by houseroad — APPROVED** [review]
This looks very legit to me. Thanks for fixing the problems. Now the execution of these tests are reproducible.

Nit: switch the order of parameters.
For example:
test_log(self, n, m, gc, dc, seed) ==> test_log(self, n, m, seed, gc, dc)

**Pull request merged**